xmtp_mls/src/storage/encrypted_store/mod.rs

```diff
@@ -216,12 +216,13 @@ macro_rules! impl_fetch {
             type Key = $key;
             fn fetch(&self, key: &Self::Key) -> Result<Option<$model>, $crate::StorageError> {
                 use $crate::storage::encrypted_store::schema::$table::dsl::*;
-                Ok(self.raw_query(|conn| $table.find(key).first(conn).optional())?)
+                Ok(self.raw_query(|conn| $table.find(key.clone()).first(conn).optional())?)
```

Comment:
The `clone` is needed to support fetching by composite primary key. In this case, `find()` expects a tuple `(key1, key2)`, not a reference `&(key1, key2)` and will return a type error otherwise